### DISCUSSION TITLE
Request to add the encodedCommand encoding function of powershell

### DISCUSSION THREAD

**Discussion by tgirl7** [projectdiscovery/nuclei#3883] [General]
When you encounter a deserialization vulnerability and need to call powershell to verify the vulnerability, you often need to call powershell to request OOB

**Comment by Mzack9999** [comment]
@tgirl7 This is supported with https://github.com/projectdiscovery/nuclei/pull/2930 and will be available within subsequent releases. Templates must be signed to be executed, as invoking external commands is susceptible to code injection.
